client/js/lounge.js

```diff
@@ -994,6 +994,23 @@ $(function() {
 		});
 	});
 
+	$("#connect").one("show", function() {
+		var params = window.URI(document.location.search);
+		params = params.search(true);
+		// Possible parameters:  name, host, port, password, tls, nick, username, realname, join
+		for (var key in params) {
```

Comment:
`for (var ... in ...)` includes properties that are inherited through the prototype chain, and this is generally not what you want. See http://eslint.org/docs/rules/guard-for-in for an alternative.
Also https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in#Iterating_over_own_properties_only.